README.md

```diff
@@ -65,7 +65,7 @@ end
 
 ## Installation
 
-1. Add `require "active_storage"` to config/application.rb.
+1. Add `require "active_storage"` to config/application.rb, after `require "rails/all"` line.
```

Comment:
Maybe it could even be `require "active_storage/railtie"` to be consistent with what Rails generates whenever you opt-out from a framework like Active Record or Action Mailer. What do you think ?